app/containers/LoginLedgerNanoS/LoginLedgerNanoS.jsx

```diff
@@ -106,6 +164,74 @@ export default class LoginLedgerNanoS extends React.Component<Props, State> {
     )
   }
 
+  fetchAdditionalKeys = async () => {
+    const { publicKeys } = this.state
+
+    this.setState({ loadingPublicKeys: true, error: null })
+
+    const lastAccountLoaded = publicKeys[publicKeys.length - 1].account
+
+    const nextBatchOfKeys = await getPublicKeys(lastAccountLoaded + 1).catch(
```

Comment:
From this I deduce that `account` is an array index. If so, I think it should be indicated in the key name (e.g `publicKeys[0].accountIdx`).

Reply:
Hmm I see what you are saying... `account` is the var name used throughout the other ledger files - I can update all the other instances if you think its better

Reply:
Leave it be them. Let's not go crazy 🤟🏻